@xen-orchestra/lite/src/components/TabConsole.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+import Console from './Console'
+import RangeInput from './RangeInput'
+
+interface ParentState {}
+
+interface State {
+  consoleSize: {
+    height: number
+    width: number
+  }
+}
+
+interface Props {
+  vmId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {
+  scaleConsole: (value: number) => void
+}
+
+interface Computed {}
+
+const TabConsole = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    initialState: () => ({
+      consoleSize: {
+        height: 768,
+        width: 1024,
```

Comment:
Why these specific values?

Reply:
This is the default size of the canvas from `novnc`

Reply:
It doesn't have a default size, it has the same dimensions as the VM screen resolution.

Reply:
Ho okay. And the VM screen resolution can be different on each VM ?

Reply:
Yes, it can be anything at all depending on the OS settings.